plugins/executor/python/server.py

```diff
@@ -0,0 +1,74 @@
+import json
```

Comment:
Should we expect plugin authors to write some tests?

Reply:
good question - we should discus this - I think we should expect quality - band that may need tests - what kind of tests?